ctfo_server.h

```diff
@@ -158,18 +169,21 @@ class CTFOServer final {
   using Storage = CTFOStorage<SherlockStreamPersister>;
   using MidichloriansServer = current::midichlorians::server::MidichloriansHTTPServer<CTFOServer>;
   using RESTStorage = RESTfulStorage<Storage, CTFOHypermedia>;
-
-  explicit CTFOServer(const CTFOServerParams& params)
-      : port_(params.api_port),
-        midichlorians_server_(
-            params.GetMidichloriansPort(), *this, params.tick_interval_ms, "/ctfo/log", "OK\n"),
-        debug_print_(params.debug_print_to_stderr),
-        storage_(current::sherlock::SherlockNamespaceName("OldCTFOStorage"), params.storage_file),
-        rest_(storage_, params.GetRESTPort(), "/ctfo/rest", params.rest_url_prefix) {
-    midichlorians_stream_.open(params.midichlorians_file, std::ofstream::out | std::ofstream::app);
+  using Config = current::SelfModifyingConfig<CTFOServerParams>;
+
+  explicit CTFOServer(const std::string& config_path)
+      : config_(config_path),
+        midichlorians_server_(config_.Config().GetMidichloriansPort(),
+                              *this,
+                              config_.Config().tick_interval_ms,
+                              "/ctfo/log",
```

Comment:
Then we need to name this parameter, too =). @sompylasar , @dkorolev , @mzhurovich , what do you think? `midichlorians_url_path` / `mirichlorians_http_route` ?